cmd/perftester/main.go

```diff
@@ -0,0 +1,138 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+	"strconv"
+	"time"
+)
+
+func main() {
+	times, err := strconv.Atoi(os.Args[1])
+	if err != nil {
+		fmt.Println("Wrong number:", err)
+		os.Exit(1)
+	}
+
+	err = os.Chdir("../..")
+	if err != nil {
+		fmt.Println("Error changing the directory:", err)
+		os.Exit(1)
+	}
+
+	for i := 0; i < times; i++ {
+		makeIteration()
+		time.Sleep(10 * time.Second)
+	}
+}
+
+func makeIteration() {
```

Comment:
you could use signature `makeIteration() error` and just return error from here, then do logging and `os.Exit` on the caller's side